src/renderer/components/App.tsx

```diff
@@ -50,11 +50,13 @@ export default class App extends React.Component<any, undefined> {
 
     // Called when files are droped on the dropzone
     public onDrop(acceptedFiles: File[]) {
+        const tabfile: File[] = this.cleanTabFile(acceptedFiles);
+
         this.store.dispatch(
             dialogActions.open(
                 DialogType.FileImport,
                 {
-                    files: acceptedFiles.map((file) => {
+                    files: tabfile.map((file) => {
```

Comment:
This is a minor coding remark: instead of introducing a new function `cleanTabFile()`to filter out some values from the original array of dropped files, you could simply use the `Array.filter()` function with a simple function predicate that excludes non-recognized file extensions, and chain the result (`acceptedFiles.filter(...).map(...)`). See:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter